packages/pipeline-editor/src/PipelineService.tsx

```diff
@@ -27,6 +27,8 @@ export const RUNTIMES_NAMESPACE = 'runtimes';
 export const RUNTIME_IMAGES_NAMESPACE = 'runtime-images';
 export const COMPONENTS_NAMESPACE = 'components';
 
+const ELYRA_PREFIX_LEN = 6;
```

Comment:
Could we add a comment here indicating what this is about?  Perhaps something like: 
```
/*
 * Generic component properties are prefixed with 'elyra_' in order to distinguish themselves from custom component properties.
 */
```

This kind of comment could go at the point of use instead.

@kiersten-stokes - is that wording correct?

Reply:
I haven't pushed my updates yet, but this line will be gone and replaced with a util function with a comment similar to the one you gave

Reply:
That sounds good! If we wanted to be more specific, maybe something like "Generic component properties and certain meta-properties (e.g., 'label') that apply to both generic and custom components are prefixed..." and so on as above